Kernel/Scheduler.cpp

```diff
@@ -523,6 +546,38 @@ void Scheduler::initialize()
     // Make sure the colonel uses a smallish time slice.
     s_colonel_process->set_priority(Process::IdlePriority);
     load_task_register(s_redirection.selector);
+
+    g_timer_queue = new SinglyLinkedList<Timer>();
+
+    auto timer = Timer();
+    timer.expires = g_uptime + 10000;
+    timer.callback = []() {
+	dbgprintf("10 secs\n");
+    };
+    add_timer(timer);
```

Comment:
I would suggest adding some convenience helpers that would allow us to write code like:

```cpp
add_timer(10, [] {
    dbg() << "10 secs";
}):
```

Note that we'll need to use the `TICKS_PER_SECOND` mutliplier rather than hard-coding `1000`, or we'll break the ability to change the kernel's tick frequency.